src/Host/Client/Impl/BrokerServices/ISessionsWebService.cs

```diff
@@ -2,13 +2,13 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System.Collections.Generic;
+using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.R.Host.Protocol;
 
 namespace Microsoft.R.Host.Client.BrokerServices {
     public interface ISessionsWebService {
-        Task<IEnumerable<SessionInfo>> GetAsync();
-
-        Task<SessionInfo> PutAsync(string id, SessionCreateRequest request);
+        Task<IEnumerable<SessionInfo>> GetAsync(CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
The method names were supposed to correspond to controller endpoints in the broker. So if it's `/sessions` and `SessionsController`, then this side has `SessionsWebService`. Since it has a `GetAsync` method for when you access the path directly, there's one here too. For something like `/sessions/foo`, it would be `GetFooAsync` etc.

I'm not really married to this scheme, though. It's all down to what's more convenient to use. If multiple classes start to get in the way, let's change it.